requirements.txt

```diff
@@ -2,7 +2,7 @@ async-generator==1.10
 attrs==21.4.0
```

Comment:
🏕️ If time permits we could change this to `pip-compile` :) but maybe not in this PR.